pkg/deployment/deployment_item.go

```diff
@@ -47,6 +48,55 @@ type DeploymentItem struct {
 	renderedYamlPath      string
 }
 
+// TODO: find better name then DeploymentItem...Vars?
+// IDEA: Make these sub-objects of actual di/dp
+type DeploymentItemProjectVars struct { // info about the deployment.yaml project that included this item
+	Parent  *DeploymentItemProjectVars // <parent project info, same structure as `project`>
+	DirName string                     `yaml:"dirName" validate:"required"` // dirname
+	RelPath []string                   // <relative directory to current source>
+	//RelRoot string                     // rel path to the project root dir, ie: ../../..
+	Tags *utils.OrderedMap // all tags, including the ones merged from the parents
+}
+type DeploymentItemVars struct {
+	DirName string                      `yaml:"dirName" validate:"required"` // dirname
+	DirPath string                      `yaml:"dirPath,omitempty"`           // full path to deploymentItem dir
+	RelRoot string                      `yaml:"relRoot,omitempty"`           // relative to project root dir
+	RelPath []string                    `yaml:"relPath,omitempty"`           // relative from project root, <relative directory to current source> (the "source" might differ in case we enter a gitInclude)
+	RevPath []string                    `yaml:"revPath,omitempty"`           // relative to project root
```

Comment:
Is this the reverse of the path? I'd suggest to not include this as it can easily be achieved via Jinaj2 templating (`reverse(this.relPath)`)

Reply:
well I sort of agree, but read the above comment pls.

Reply:
no really `reverse` is in jinja an `filter`. Cant be used as you suggest. Other limitation is that there is no filter like `index(1)` .. (to my surprise). 

reverse filter reverse the object or in many cases returns a in iterator